src/dart-2.0.md

```diff
@@ -104,12 +111,13 @@ As the example shows, you can use the static `dynamic` type
 to indicate that the runtime type is unknown.
 One big difference between optionally typed Dart and Dart 2.0
 is that, in the former, the analyzer infers the `dynamic` type.
-In Dart 2.0, both the analyzer and
-the runtime try to infer something more specific than `dynamic`.
+In Dart 2.0, tools like the analyzer can infer
+more specific types than `dynamic`,
+following rules in the language specification.
 
-While Dart 2.0 is statically typed, type inference and types like
-`dynamic` and `num` (which can be `int` or `double`)
-take away much of the burden of specifying types.
+While Dart 2.0 is statically typed, type inference and
+less precise types such as `dynamic` and `num`
+take away much of the burden of specifying exact types.
```

Comment:
Why not simply:
> While Dart 2.0 is statically typed, type inference takes away much of the burden of writing type annotations.

Reply:
I think we wanted to let people know that they could be specifically unspecific... that if they knew the type could be any object or any number, they could indicate it.

Reply:
Ok (though that which takes away the burden is the type inference).